src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -55,14 +61,16 @@ public function load($data, $type = null)
 
         foreach ($this->resourceNameCollectionFactory->create() as $resourceClass) {
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            $normalizedShortName = Inflector::pluralize(Inflector::tableize($resourceMetadata->getShortName()));
+
+            $shortNamePath = $resourceMetadata->getShortName() ?? '';
```

Comment:
What do you mean by passing the `$resourceShortName ` into `addRoute` ? The argument passed to `namePath` is still null :/